scrapyrt/log.py

```diff
@@ -64,7 +64,7 @@ def _unicode_to_str(self, eventDict):
         message = eventDict.get('message')
         if message:
             eventDict['message'] = tuple(
-                unicode_to_str(x, self.encoding) for x in message)
+                to_bytes(x, self.encoding) for x in message)
```

Comment:
this is good point, maybe we should add tests in tox for different scrapy versions what do you think?